test/extern/unions/externUnionRecordErrorLLVM.skipif

```diff
@@ -0,0 +1 @@
+CHPL_TARGET_COMPILER!=llvm
```

Comment:
Why are we only testing this error for LLVM?

Reply:
Because if we aren't using the LLVM backend we can't know if the `extern record` is declared as a C `struct` or `union`.